src/renderer_mtl.mm

```diff
@@ -1294,6 +1298,32 @@ void blitSetup(TextVideoMemBlitter& _blitter) override
 		{
 			BX_UNUSED(_blitter);
 		}
+	
+#if BX_PLATFORM_VISIONOS
+		void calculateViewPorts(MTLViewport (&viewports)[2]) {
+			const int viewCount = 2;
+			for (int i = 0; i < viewCount; i++) {
+				cp_view_t view = cp_drawable_get_view(m_mainFrameBuffer.m_swapChain->m_drawable, i);
+				cp_view_texture_map_t texture_map = cp_view_get_view_texture_map(view);
+				viewports[i] = cp_view_texture_map_get_viewport(texture_map);
+			}
+		}
+		
+		void setVertexAmplification(RenderCommandEncoder rce) {
```

Comment:
is this supposed to make a deep copy on call? seems like we'd want it to be a reference.
```suggestion
		void setVertexAmplification(RenderCommandEncoder& rce) {
```

Reply:
Good catch!